fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContext.java

```diff
@@ -926,7 +926,7 @@ private <T> void addAnonymousCommitHookToMap(@Nonnull Map<String, T> map, @Nonnu
             String name;
             // Yes, a collision is exceedingly unlikely, but...
             do {
-                name = INTERNAL_COMMIT_HOOK_PREFIX + "anon-" + (new Random()).nextInt(Integer.MAX_VALUE);
+                name = INTERNAL_COMMIT_HOOK_PREFIX + "anon-" + ThreadLocalRandom.current().nextInt(Integer.MAX_VALUE);
```

Comment:
This is to fix a new SpotBugs issue. It doesn't like a new random being created each time.